src/subdomains/core/payment-link/dto/payment-link.dto.ts

```diff
@@ -1,39 +1,39 @@
 import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
 import { Blockchain } from 'src/integration/blockchain/shared/enums/blockchain.enum';
 import { FiatDto } from 'src/shared/models/fiat/dto/fiat.dto';
+import { PaymentLinkPaymentMode, PaymentLinkPaymentStatus, PaymentLinkStatus } from '../enums';
 
-export enum PaymentLinkStatus {
-  ACTIVE = 'Active',
-  INACTIVE = 'Inactive',
-}
+export type TransferMethod = Blockchain;
 
-export enum PaymentLinkPaymentStatus {
-  PENDING = 'Pending',
-  COMPLETED = 'Completed',
-  CANCELLED = 'Cancelled',
-  EXPIRED = 'Expired',
+export interface TransferInfo {
+  asset: string;
+  amount: number;
+  method: TransferMethod;
+  quoteUniqueId: string;
 }
 
-export enum PaymentLinkPaymentMode {
-  SINGLE = 'Single',
-  MULTIPLE = 'Multiple',
+export interface TransferAmount {
+  method: TransferMethod;
+  gasPrice: number;
+  assets: TransferAmountAsset[];
 }
 
-export type TransferMethod = Blockchain;
-
-export interface TransferInfo {
+export interface TransferAmountAsset {
   asset: string;
   amount: number;
-  method: TransferMethod;
 }
 
+export type RequestedAmountAsset = TransferAmountAsset;
+
 export interface PaymentLinkPayRequestDto {
   tag: string;
   callback: string;
   minSendable: number;
   maxSendable: number;
   metadata: string;
-  transferAmounts: TransferInfo[];
+  quoteId: string;
+  requestedAmount: RequestedAmountAsset;
```

Comment:
Maybe like this?
```ts
quote: {
  id: string,
  expiration: Date,
}
```

Reply:
Add `displayName`

Reply:
What do you mean with "Add displayName"?